src/fusor/harvester.py

```diff
@@ -95,36 +110,70 @@ async def load_records(
         :raise ValueError: if the file does not exist at the specified path
         :return: A list of translated fusions, represented as AssayedFusion objects
         """
+        records = await self.load_record_table(fusion_path)
+        fusions = [
+            record.annotated
+            for record in records
+            if record.annotated is not None
+            and isinstance(record.annotated, AssayedFusion)
```

Comment:
We could simplify this

```suggestion
            if isinstance(record.annotated, AssayedFusion)
```